src/inspect.rs

```diff
@@ -144,6 +144,7 @@ impl MetadataPrinter {
             }
         }
         table.add_row(row![Fgbl -> "mutating:", metadata.mutating]);
+        table.add_row(row![Fgbl -> "backgroundAudit:", metadata.background_audit]);
```

Comment:
Maybe print something more human friendly: `background audit support:`

Reply:
true, better! done in https://github.com/kubewarden/kwctl/pull/339/commits/fcdf8db14c898ac4c6f49800b1fcb183dda79440.